cypress/integration/precise-rounding.js

```diff
@@ -0,0 +1,395 @@
+                            baseline,
+                            growth,
+                            progress,
+                            performanceLevel
+                        });
+                    });
+                });
+        });
+  };
+
+    // CSV Tests
+    const checkCSVDataAgainstDocsTestData = ({student, contentArea, competency, baseline, growth, progress, performanceLevel} ) => {
+        cy.loginAs('admin');
+        cy.visit('/exports');
+
+        // prepare for form submission that returns back a file
+        // https://on.cypress.io/intercept
+        cy.intercept({ pathname: '/exports/slate-cbl/student-competencies'}, (req) => {
+            req.redirect('/exports')
+        }).as('records');
+
+        cy.get('form[action="/exports/slate-cbl/student-competencies"]').within(() => {
+            cy.get('input[name=students]').type(`{selectall}{backspace}${student}`);
+            cy.get('select[name=content_area]').select(contentArea);
+            cy.get('select[name=level]').select('highest');
+            cy.root().submit();
+        });
+
+        cy.wait('@records').its('request').then((req) => {
+        cy.request(req)
+        .then(({ body, headers }) => {
+            expect(headers).to.have.property('content-type', 'text/csv; charset=utf-8')
+            return csvtojson().fromString(body)
+        }).then((records) => {
+
+            const studentCompetencyRow = records.filter((record)=> {
+                return record.Competency === `${contentArea}.${competency}`
+            });
+
+            const expectedPerformanceLevel = +studentCompetencyRow[0]['Performance Level']
+            const expectedGrowth =  +studentCompetencyRow[0]['Growth']
+            const expectedBaseLine = +studentCompetencyRow[0]['Baseline']
+            const expectedProgress = +studentCompetencyRow[0]['Progress']
+
+            expect(expectedPerformanceLevel,
+            `${contentArea}.${competency} for ${student}  CSV Performance Level Value ${expectedPerformanceLevel}: Test Doc Perfomance Level Value ${performanceLevel}`
+            ).to.equal(performanceLevel);
+
+            expect(expectedGrowth,
+                `${contentArea}.${competency} for ${student} CSV Growth Value ${expectedGrowth}: Test Doc Growth Value ${growth}`
+            ).to.equal(growth);
+
+            expect(expectedBaseLine,
+                `${contentArea}.${competency} for ${student} CSV Baseline Value ${expectedBaseLine}: Test Doc Baseline Value ${baseline}`
+            ).to.equal(baseline);
+
+            expect(expectedProgress,
+                `${contentArea}.${competency} for ${student} CSV Completion Percentage Value ${expectedProgress}: Test Doc Completion Percentage Value ${progress}`
+                ).to.equal(progress/100); // progress is represented as decimal in export
+        });
+        });
+    };
+
+
+  it('Student 5 ELA Competency 7', () => {
```

Comment:
**suggestion (non-blocking):** We generally try not to get too engineer-y with tests, but this might be a good case for generating all these `it()` blocks within a loop through all the test cases loaded from a fixture file as I mentioned above. You might just put all the statements as a property inside there, like:

```json
[
  {
    "caseTitle": "Student 5 ELA Competency 7",
    "student": "student4",
    "contentArea": "ELA",
    "competency": "7",
    "baseline": "6",
    "growth": "0.4",
    "progress": "63",
    "performanceLevel": "6.4"
  },
  {
    // ...
  }
]
```